### PR TITLE
Updated with changes from original code base, fully migrated to react-router-dom-v5-compat

### PR DESCRIPTION
**Pull request by auan369**
Have included the most recent commits to the original code base and have migrated fully to react-router-dom-v5-compat

### CODE REVIEW

File: packages/web/src/pages/commanders/commandersList.tsx [hunk 1]
```diff
@@ -19,7 +20,8 @@ export const CommandersList = () => {
   }>();
 
   useEffect(() => {
-    document.title = `${commanderBase} - ${capitalize(race)}`;
```
Comment (petrvecera) [inline packages/web/src/pages/commanders/commandersList.tsx:22]: Is there any reason for this change?

**Review by petrvecera — COMMENTED** [review]
Looks great, I will test it :)

**Comment by petrvecera** [comment]
Hi @auan369 it looks like there is some issue, it doesn't load for me 

image

File: packages/web/src/pages/commanders/commandersList.tsx (hunk 1 above)
Comment (auan369) [reply packages/web/src/pages/commanders/commandersList.tsx:22]: I had issues with the capitalize during the migration. If the original line is kept, I would face this error.
Screenshot 2024-10-19 at 4 45 50 PM

Comment (petrvecera) [reply packages/web/src/pages/commanders/commandersList.tsx:22]: I see, you can fix this by passing there empty string like this ${capitalize(race || "")}

**Comment by auan369** [comment]
Tr

> Hi @auan369 it looks like there is some issue, it doesn't load for me
> 
> image

Trying it out on my side again and I have the same issue now as well. Seems like issue with Link in main-header.tsx and main-footer.tsx still persists. So changing the import to

**Comment by auan369** [comment]
Hi @petrvecera , I have discovered the issue with the Links. The update is to shift the CompatRouter component the above MainHeader and MainFooter components. Then we can use react-router-dom-v5-compat for all files.
Screenshot 2024-10-19 at 5 06 19 PM

**Comment by petrvecera** [comment]
> Hi @petrvecera , I have discovered the issue with the Links. The update is to shift the CompatRouter component the above MainHeader and MainFooter components. Then we can use react-router-dom-v5-compat for all files. 

Nice, can you push this change? 👍

**Comment by auan369** [comment]
Just sent the PR 👍

**Comment by petrvecera** [comment]
closing this in favor of https://github.com/cohstats/coh2stats/pull/359 

FYI: you can always just update the original branch, no need to create a new one, you can also always do git push --force to force the change in the commits

**Pull request closed without merging**